lib/crypto/sig_verifier.go

```diff
@@ -66,12 +64,12 @@ func (sv *SignatureVerifier) Start() {
 			case <-sv.closeCh:
 				return
 			default:
-				sign := sv.Remove()
-				if sign == nil {
+				sig := sv.Remove()
```

Comment:
nit can you rename it to `signature` please? I think we should aim to use full words to avoid confusion :wink: Thanks!